### ISSUE TITLE
load overlay in command line option?

### ISSUE THREAD

**Issue by dreamsoftunitedkingdom** [libretro/RetroArch#12370]
hi

I found you can load a specifc core, game rom and cfg file from the command line in windows (passing to retroarch).

I was wondering if it's possible to load a game rom and a specific overlay file for that game (copied in overlays folder of retroarch) on the command line and if yes, how to do this?

thx

**Comment by hizzlekizzle** [comment]
No, the closest you can do is --appendconfig and point to a config fragment that calls out the overlay. It's an extra bit of faff vs just calling it out directly, but not too bad.

**Comment by dreamsoftunitedkingdom** [comment]
Can you give an example that uses lets say rom.zip as game passed to command line?
How/what I need to do exactly for passing the overlay file?

I'm a beginner in retroarch and don't find any docs about this.

So my goal is to run a game with specific overlay file by using the command line.(in windows)

thx

**Comment by hizzlekizzle** [comment]
So, if you have a text file named foo.cfg with contents:
```
input_overlay = "C:\path\to\overlay.cfg"
config_save_on_exit = "false"
```
You can launch RetroArch like this:
`.\retroarch.exe --appendconfig=foo.cfg -L C:\path\to\core.dll C:\path\to\ROM.ext`
and it should apply the overlay from foo.cfg
